config/routes.rb

```diff
@@ -5,15 +5,18 @@
   root 'home#index'
 
   resources :cards
-  resources :users
-  resources :user_sessions #, only: [:new, :create, :destroy]
+  resources :users, except: [:new, :create]
+  resources :registration, only: [:new, :create]
+  resources :user_sessions
+
+  post '/users', to: 'registration#create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.